### PR TITLE
lightning-specification: add 23/01/30 meeting transcript

### PR DESCRIPTION
**Pull request by carlaKC**
Generated transcript for LN spec meeting on 23/01/30. 

Done one human-checking pass. 

Opening as a draft for now because it needs one more cleanup round before it's worth reviewing.

### CODE REVIEW

**Review by gurwindersahota — COMMENTED** [review]
First pass- Probably missed a few things here and there.

File: lightning-specification/2023-01-30-specification-call.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+
+Speaker 0: Cause if you have zero there, zero greater than zero false. So. 
+
+Speaker 1: Oh yeah, look, this is not the scariest thing in Bitcoin. Like, you know, hey, we validated it. It works. I'm like, okay, if we save one byte, you know, think of the starving children or think of the NFTs you could create in that. Yeah, the starving blocks. Yeah. I guess it's part of me that goes, shouldn't we just be generating all these things from mini script these days and doing that? And what does it generate? Yeah, so yeah. So everything here has just been hand-rolled. I don't think there's been mini script. Maybe that, maybe mini script popped this out. 
+
+Speaker 0: I'm not sure. But like, so initially I had this little version and then the contributor, I think it was {redacted} came through and then gave us the stuff which is a little bit smaller. And then I think it's just like, I don't know, I don't know. I don't know if it's like, I don't know if it's like, I don't know if it's like, I don't know. I don't know if it's like, can we plug that in and get what, you know, something like that.
```
[the remark this answers is not in the document]
Comment (carlaKC) [reply lightning-specification/2023-01-30-specification-call.md:165]: No, I'm sure one is enough!

File: lightning-specification/2023-01-30-specification-call.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+
+Speaker 0: I mean, so I guess y'all take me into basically like there's new commitments. There's new TLV equipment segment that actually has multiple signatures for each of the pending splices. But then, then you also have that for all the HTLCs as well, because one might get very large potentially or like run into potential like issues as far as the smallest message size or lunch message size rather.
```
[the remark this answers is not in the document]
Comment (carlaKC) [reply lightning-specification/2023-01-30-specification-call.md:54]: I think it's something like - "can you talk me through"?

**Comment by carlaKC** [comment]
Thanks for the detailed review @gurwindersahota! 

I've included the suggestions + done one more pass on the techincal terminology - diff is [here](https://github.com/bitcointranscripts/bitcointranscripts/compare/789aa3a1cfadba25f92945977c5ab29423dc0d45..fe4327950b22c6bab762e8bd7dee1cf00dfd7b0f).

**Comment by adamjonas** [comment]
@carlaKC can you take this out of draft mode so we can merge it?

**Pull request merged**